Userland/Shell/Tests/test-commons.inc

```diff
@@ -1,11 +0,0 @@
-fail() {
-    # The AST ends up at stderr, and CI checks for 'fail' (uppercase)
-    # Trick it into not failing the test.
-    echo "FA""IL:" $*
-    exit 1
-}
-
-pass() {
-    echo "PA""SS"
-    exit 0
-}
```

Comment:
why is this removed?

Reply:
Not sure why... I moved it to the new directory but the diff says otherwise :man_shrugging: